packages/cli/src/commands/gsn-relayer-withdraw.ts

```diff
@@ -0,0 +1,44 @@
+import { CommandsLogic, WithdrawOptions } from '../CommandsLogic'
+import { gsnCommander, getKeystorePath, getServerConfig } from '../utils'
+import { createCommandsLogger } from '../CommandsWinstonLogger'
+import { KeyManager } from '@opengsn/relay/dist/KeyManager'
+import { toBN } from 'web3-utils'
+
+const commander = gsnCommander(['f', 'g'])
+  .option('-k, --keystore-path <keystorePath>', 'relay manager keystore directory', '/app/data/gsndata/manager/')
```

Comment:
please assume default path is ` -v $PWD:$PWD -w $PWD`, and then the default path should be `./gsndata/manager`

Reply:
Was already fixed...